lib/psyduck.rb

```diff
@@ -11,24 +11,32 @@ class FTPClient
     attr_reader :passivity
     attr_reader :resumable
 
-    def initialize(ip_address, username = 'anonymous', password = nil, command_port = 21, passivity = true)
+    def initialize(ip_address, username = 'anonymous', password = nil, command_port = 21, passivity = true, options = {})
```

Comment:
I think we'd be better off with keyword args here... not obvious what order the args are going in